docs/src/rsh/consensus/index.md

```diff
@@ -148,6 +148,33 @@ if `{!rsh} COND_EXPR` is true, then the block executes,
 and if not, then the loop terminates and control transfers to the continuation of the while statement.
 The identifiers bound by `{!rsh} LHS` are bound within `{!rsh} DEFINE_BLOCK`, `{!rsh} INVARIANT_EXPR`, `{!rsh} COND_EXPR`, `{!rsh} BLOCK`, and the tail of the while statement.
 
+```reach
```

Comment:
You should sample this from a file

Reply:
In our DCT team meeting yesterday, we discussed how we could ensure that the coding samples did not get modified and change the line numbers that we referenced in the examples folder. So Njoku is creating a doc-samples folder to be inside of the examples folder with a document where we can put all of the samples we use with a section for each term in alphabetical order with room for 15 lines. However, the issue was brought up about what if the coding sample is longer than 15 lines. For that, it was decided that we would place the few code samples that are longer than 15 lines directly to the document. This code sample is around 20 lines, so I put it directly in the document.